### PR TITLE
[O2B-1092] add migration file

### PR DESCRIPTION
**Pull request by xsalonx**
#### I have a JIRA ticket
- [x] branch and/or PR name(s) include(s) JIRA ID
- [ ] issue has "Fix version" assigned
- [x] issue "Status" is set to "In review"
- [x] PR labels are selected

Notable changes for users:
- NA

Notable changes for developers:
- NA

Changes made to the database:
- add migration file for data passes table creation

### CODE REVIEW

File: lib/database/migrations/20231211080706-create-data-passes.js [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+                type: Sequelize.STRING,
+                allowNull: false,
+            },
+            description: {
+                type: Sequelize.TEXT,
+            },
+            output_size: {
+                type: Sequelize.BIGINT,
+            },
+            reconstructed_events: {
```
Comment (martinboulais) [inline lib/database/migrations/20231211080706-create-data-passes.js:23]: For this I would put a `..._count` if it's a counter

File: lib/database/migrations/20231211080706-create-data-passes.js [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+                type: Sequelize.INTEGER,
+            },
+            last_run: {
```
Comment (martinboulais) [inline lib/database/migrations/20231211080706-create-data-passes.js:26]: Here I would put `last_run_number`

**Review by martinboulais — CHANGES_REQUESTED** [review]

File: lib/database/migrations/20231211080706-create-data-passes.js (hunk 1 above)
Comment (xsalonx) [reply lib/database/migrations/20231211080706-create-data-passes.js]: done

File: lib/database/migrations/20231211080706-create-data-passes.js (hunk 2 above)
Comment (xsalonx) [reply lib/database/migrations/20231211080706-create-data-passes.js]: done

**Review by martinboulais — APPROVED** [review]

**Pull request merged**